capstone-frontend/src/App.jsx

```diff
@@ -24,6 +25,7 @@ class App extends Component {
     this.state = {
       searchQuery: "",
       bookLists: [],
+      isSignIn: ((window.localStorage.getItem("userID")) && (window.localStorage.getItem("profileObj"))) ? true : false,
```

Comment:
Couldn't this be just the boolean part of the ternary expression? Doing <boolean> ? true : false will evaluate to <boolean> anyways.

Reply:
`nit`: Also isSignedIn might be a better name

Reply:
Changed Name. I used a ternary expression to make sure the state is not initialize to a String